modules/images/webp-uploads/load.php

```diff
@@ -742,3 +742,29 @@ function webp_uploads_get_image_sizes_additional_mime_type_support() {
 
 	return $allowed_sizes;
 }
+
+/**
+ * Updates the quality of images used in the wp editor.
+ * For WP version less than 6.1 or WebP returns 82,
+ * Otherwise returns default quality.
+ *
+ * @since n.e.x.t
+ *
+ * @global string $wp_version The WordPress version string.
+ *
+ * @param int    $quality   Quality level between 1 (low) and 100 (high).
+ * @param string $mime_type Image mime type.
+ * @return int The updated quality for mime types.
+ */
+function webp_uploads_set_wp_image_editor_quality( $quality, $mime_type ) {
+	global $wp_version;
+
+	// Below WP 6.1 or WebP, always return 82.
+	if ( version_compare( $wp_version, '6.1', '<' ) || 'image/webp' === $mime_type ) {
+		return 82;
+	}
+
+	// Return default quality for WP 6.1+.
```

Comment:
This comment is misleading: The code doesn't always return the default quality for WP 6.1+, it only returns the default quality if it's WP 6.1+ _and_ the image is not WebP.
```suggestion
	// Return default quality for non-WebP images in WP 6.1+.
```